src/app/footer/footer.component.ts

```diff
@@ -78,6 +80,7 @@ export class FooterComponent extends Base implements OnInit {
               this.version = metadatum;
             }
             this.content = this.footerService.versionsToMarkdown(
+              this.domain,
```

Comment:
You can use `DOCKSTORE.HOSTNAME`. It already does the calculation, and it will just return `https://dockstore.org`, whereas `window.location.href` will include the current path based on where you are, e.g., if you are on the orgs page, `https://dockstore.org/organizations`, which we probably don't want.